examples/with-cte/README.md

```
@@ -0,0 +1,176 @@
+  // delegation (optional):
+  actorToken: "eyJ...",
+  actorTokenType: "https://corporate-idp.example.com/id-token",
+});
+// result.accessToken, result.act, ...
+```
+
+`customTokenExchange` is **server-side only** and does NOT modify the user session. It returns tokens that the caller can use directly or store as needed.
+
+### Request flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)